src/components/policy/policy_regular/src/policy_manager_impl.cc

```diff
@@ -778,7 +779,7 @@ uint32_t PolicyManagerImpl::NextRetryTimeout() {
   ++retry_sequence_index_;
 
   for (uint32_t i = 0u; i < retry_sequence_index_; ++i) {
-    next += retry_sequence_seconds_[i];
+    next += retry_sequence_seconds_[i] * 1000;
     // According to requirement APPLINK-18244
     next += retry_sequence_timeout_;
```

Comment:
@jacobkeeler According to requirement 

>The timeout of the N retry must be count the following way:   time for_the_ (N)retry = time_ for_ the_(N-1)retry + <"seconds_between_retries">[N] + timeout
where N>=1 and retries are started to be numerated from "0"